enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: Object;
```

Comment:
It looks like the type of `running` should be `{ [name: string]: ((this: null, ...args: any[]) => void)[] }` ([code](https://github.com/webpack/enhanced-resolve/blob/7df23d64da27cd76b09046f9b9ffd61480c0ddca/lib/CachedInputFileSystem.js#L51-L56))

Or, rather, anything with an `{ apply: (thisArg: any, args?: any[]) }`, which includes `Function`s themselves as well as `Tapable.Plugin` (Tapable relies on this polymorphic behavior).